### DISCUSSION TITLE
Db admin

### DISCUSSION THREAD

**Discussion by KellerKev** [emmett-framework/emmett#349] [Ideas]
I was wondering what you think about an admin web interface to be able to quickly enter data for testing and prototyping to the models?

**Comment by gi0baro** [comment]
@Kkeller83 I think is a good idea for an extension, and there's 2 possible ways of implementing it:

- using forms directly from Emmett
- using a frontend-only app talking with standard REST endpoints built on tables using Emmett-REST

I don't have any specific preference regarding the specific solution.
I'm more of a console guy, this and my time availability combined is why I never built an extension like this

**Comment by KellerKev** [comment]
I think I can see if I can make some sort of simple crud grid based on Emmetts forms that could also be used for that. Let's see

**Comment by KellerKev** [comment]
That being said, how do I find all tables and fields of a db with the Emmett orm? I rememr in pydal it was something like tables=db.tables and then I could iterate from tables in tables and simliar to fields

**Reply by gi0baro** [reply]
@Kkeller83 yes, the `Database` class from emmett has the same property. In addition, the tables have a `_model_` attribute through which you can access the `Model` instance of the table. `Model` has a `fields` attribute you can use, and the table should have the same as well.
